modules/account-lib/test/resources/hbar/hbar.ts

```diff
@@ -1,112 +1,46 @@
 export const ACCOUNT_1 = {
   accountId: '0.0.81320',
-  publicKey: '302a300506032b65700321005a9111b5e6881ff20b9243a42ac1a9a67fa16cd4f01e58bab30c1fe611ea8cf9',
   privateKey: '302e020100300506032b65700422042062b0b669de0ab5e91b4328e1431859a5ca47e7426e701019272f5c2d52825b01',
+  publicKey: '302a300506032b65700321005a9111b5e6881ff20b9243a42ac1a9a67fa16cd4f01e58bab30c1fe611ea8cf9',
```

Comment:
mmm jorge just moved this lol - can you check in with @argjv ?

Reply:
I'll merge/rebase from master

Reply:
I did a rebase to avoid further conflicts squashing the commits later